src/model/reducers/app-navigation-action.ts

```diff
@@ -0,0 +1,29 @@
+export const SET_ACTIVE_TAB = 'setActiveTab'
+export const SET_CONNECTION_STATUS = 'setConnectionStatus'
+export const TOGGLE_SHOW_SETTINGS = 'toggleShowSettings'
+
+export function setActiveTab(tabIndex: number): {
+    type: string
+    tabIndex: number
+} {
+    return {
+        type: SET_ACTIVE_TAB,
+        tabIndex: tabIndex,
```

Comment:
You can use shorthand notation:
```suggestion
        tabIndex,
```

Reply:
I know it works, as i have seen Anders use it a couble of times. 
I however am not sure what is best for readability. 
Personally i like it as it is currently.